packages/core/src/card/register.ts

```diff
@@ -5,12 +5,12 @@
  */
 
 import { registerElementSafely } from '@cds/core/internal';
-import { CdsCardTagName, CdsCard } from './card.element.js';
+import { CdsCard } from './card.element.js';
 
-registerElementSafely(CdsCardTagName, CdsCard);
+registerElementSafely('cds-card', CdsCard);
 
 declare global {
   interface HTMLElementTagNameMap {
-    [CdsCardTagName]: CdsCard;
+    'cds-card': CdsCard;
```

Comment:
I realized why we don't export the tag names. Without the string inlined in the type it breaks several dev tools. Editors can't statically analyze and provide component types reliably. Also the web-component-analyzer can't safely generate our API spec/json.